Description: fix errors in POD. Mainly just missing =over/=back.
Origin: vendor
Bug: https://sourceforge.net/tracker2/?func=detail&aid=2556457&group_id=73194&atid=53705
Forwarded: yes
Author: Damyan Ivanov <dmn@debian.org>
Last-Update: 2011-08-05

--- a/lib/Crypt/OpenSSL/DSA/Signature.pod
+++ b/lib/Crypt/OpenSSL/DSA/Signature.pod
@@ -20,13 +20,19 @@
 
 =head1 CLASS METHODS
 
+=over
+
 =item $sig_obj = Crypt::OpenSSL::DSA::Signature->new();
 
 Create a new DSA Signature Object.  You will need to
 call set_r and set_s after you create this.
 
+=back
+
 =head1 OBJECT METHODS
 
+=over
+
 =item $r = $sig_obj->get_r;
 
 Gets first member of signature pair.
@@ -43,6 +49,8 @@ Sets first member of signature pair.
 
 Sets second member of signature pair.
 
+=back
+
 =head1 AUTHOR
 
 T.J. Mather, E<lt>tjmather@maxmind.comE<gt>
